xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -140,6 +138,14 @@ final class ClientXdsClient extends XdsClient implements XdsResponseHandler, Res
   static final int INITIAL_RESOURCE_FETCH_TIMEOUT_SEC = 15;
   private static final String TRANSPORT_SOCKET_NAME_TLS = "envoy.transport_sockets.tls";
   @VisibleForTesting
+  static final long DEFAULT_RING_HASH_LB_POLICY_MIN_RING_SIZE = 1024L;
```

Comment:
It looks like these were accidentally re-added.

Reply:
Yup, gone now.